src/testSetup/groovy/com/bmuschko/gradle/docker/TestUtils.groovy

```diff
@@ -0,0 +1,26 @@
+package com.bmuschko.gradle.docker
+
+class TestUtils {
```

Comment:
`TestUtils` sounds a bit generic. Can you break this up into `TextUtils` for the escape logic and `OsUtils` for the isWindows check?